cypress/integration/fileMenu.cy.js

```diff
@@ -166,7 +166,7 @@ describe('file menu', () => {
 
         clickMenubarUpdateButton()
 
-        const AO_NAME = `TEST ${new Date().toLocaleString()} - "saved, valid: save" state`
+        const AO_NAME = `TEST-${Date.now()}-valid-save`
```

Comment:
I like how you've set the name to reflect the test file. Maybe do the same here and in Save?